client/src/theme/styles/components/frontend/project-collection/_header.scss

```diff
@@ -0,0 +1,146 @@
+$_icon-size: 56px;
+$_hero-max-height: 340px;
+$_font-family: 'sofia-pro', 'trueno', sans-serif;
+
+.project-collection-header {
+  display: grid;
+  grid-template-areas:
+    'title'
+    'description';
+  grid-gap: 30px;
+  margin-bottom: 30px;
```

Comment:
This margin shouldn't be a property of this component, but rather of whatever component that wraps it. This might not be achievable until we have a layout component for all the frontend entities, but keep it in mind as you build in that direction. Eventually, the goal will be to have something like this:
```scss
.entity-wrapper {
  display: grid;
  grid-template-areas:
    'header'
    'filters'
    'body'
    'pagination';
  grid-gap: 30px;
}
```

Reply:
Very nice. I felt pretty stumped there, but that makes lots more sense to have a parent take care of that spacing.